api/v1/attester_types.go

```diff
@@ -0,0 +1,68 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// AttesterSpec defines the desired state of Attester
+type AttesterSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// PgpSecret defines the name of the secret to use for signing. If the secret doesn't already exist it will be created.
+	// +optional
+	PgpSecret string `json:"pgpSecret"`
+	// Policy defines the Rego policy that the attester will attest adherance to.
+	Policy string `json:"policy"`
+}
+
+// AttesterStatus defines the observed state of Attester
+type AttesterStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// CompiledPolicy defines whether or not the policy compiled correctly
+	CompiledPolicy string `json:"compiledPolicy"`
```

Comment:
maybe we should add another field here representing whether or not the attester is valid?  just a boolean that the controller can check when evaluating enforcers

Reply:
I'll add that in right now, this is probably going to change in the future, I only added CompiledPolicy so I had a status that I could try to update